src/unittest/test_noise.cpp

```diff
@@ -44,15 +50,42 @@ static TestNoise g_test_instance;
 
 void TestNoise::runTests(IGameDef *gamedef)
 {
+	TEST(testNoise2dAtOriginWithZeroSeed);
+	TEST(testNoise2dWithMaxSeed);
+	TEST(testNoise2dWithFunPrimes);
 	TEST(testNoise2dPoint);
 	TEST(testNoise2dBulk);
+	TEST(testNoise3dAtOriginWithZeroSeed);
+	TEST(testNoise3dWithMaxSeed);
+	TEST(testNoise3dWithFunPrimes);
 	TEST(testNoise3dPoint);
 	TEST(testNoise3dBulk);
 	TEST(testNoiseInvalidParams);
 }
 
 ////////////////////////////////////////////////////////////////////////////////
 
+void TestNoise::testNoise2dAtOriginWithZeroSeed()
+{
+	float actual{ noise2d(0, 0, 0) };
+	constexpr float expected{ -0.281791f };
```

Comment:
Why curly brackets?
```suggestion
	float actual = noise2d(0, 0, 0);
	constexpr float expected = -0.281791f;
```
same for the other test functions.

Reply:
This was written before my policy to undo uniform initialization for short commits. Uniform initialization is better, especially for floats because it won't allow initializing with a double value (implicit narrowing conversion) and it's consistent for all initialization.

I'm happy to make this change, but in the future I intend to fight for it because there's nothing in the style guide forbidding uniform initialization, and there is a semantic difference.

Reply:
I think if the uniform initialization syntax wasn't so ugly you'd see way less resistance :wink: